polkit.yaml

```diff
@@ -0,0 +1,76 @@
+package:
+  name: polkit
+  version: "126"
+  epoch: 0
+  description: "A toolkit for defining and handling authorizations."
+  copyright:
+    - license: LGPL-2.0-or-later
+  dependencies:
+    runtime:
+      - dbus
+      - linux-pam
+      - shadow
+      - systemd
+      - wolfi-base
+  scriptlets:
+    post-install: |
+      #!/bin/sh
+      systemd-sysusers
+      chgrp -R polkitd /etc/polkit-1/rules.d
```

Comment:
Scripts are not executed during image builds, thus ensure there is tmfiles.d snippet to the chgrp effect. Didn't check if there is one already.

Also we try not to use scriplets at all. Despite the interactive testing convenience.

Reply:
Hmm nevermind, it somehow seems to just work without it. So yeah, removed the post-install scriptlet.